### ISSUE TITLE
The inactive locales select needs a label

### ISSUE THREAD

**Issue by afercia** [swissspidy/preferred-languages#14]
The inactive locales select misses an associated label. It can also be a hidden one (with screen-reader-text). As per the WP a11y coding standards, the label should be explicitly associated with for/id attributes.

<img width="718" alt="screen shot 2017-05-20 at 19 14 22" src="https://cloud.githubusercontent.com/assets/1682452/26277847/d450ed68-3d90-11e7-811e-27a6dc5f270f.png">

Worth noting `wp_dropdown_languages()` prints out empty `name` and `id` attributes if the related arguments are not explicitly passed: `<select name="" id="">`, which doesn't look right to me. This should be fixed upstream.
